src/legacy/core_plugins/vis_type_timeseries/common/agg_lookup.js

```diff
@@ -127,25 +127,3 @@ const byType = {
 export function isBasicAgg(item) {
   return _.includes(Object.keys(byType.basic), item.type);
 }
-
-export function createOptions(type = '_all', siblings = []) {
```

Comment:
@alexwizp Is this unused function just a leftover or a regression that should actually be included somewhere?

Reply:
Hm... looks like we don't need it anymore.